### ISSUE TITLE
Exclusive access warnings in Swift 4.1

### ISSUE THREAD

**Issue by alanzeino** [httpswift/swifter#291]
```

/Users/alan/repositories/ios/vendor/swifter/Sources/Socket.swift:51:42: error: overlapping accesses to 'addr', but modification requires exclusive access; consider copying to a local variable
        return try withUnsafePointer(to: &addr) { pointer in
                                         ^~~~~
/Users/alan/repositories/ios/vendor/swifter/Sources/Socket.swift:59:72: note: conflicting access is here
                return Int(OSHostByteOrder()) != OSLittleEndian ? addr.sin_port.littleEndian : addr.sin_port.bigEndian
                                                                  ~~~~~^~~~~~~~
/Users/alan/repositories/ios/vendor/swifter/Sources/Socket.swift:66:42: error: overlapping accesses to 'addr', but modification requires exclusive access; consider copying to a local variable
        return try withUnsafePointer(to: &addr) { pointer in
                                         ^~~~~
/Users/alan/repositories/ios/vendor/swifter/Sources/Socket.swift:71:31: note: conflicting access is here
            return Int32(addr.sin_family) == AF_INET
                         ~~~~~^~~~~~~~~~
```

Xcode 9.3b2

**Comment by ignatovSA** [comment]
These warnings are compilations error in Xcode 10.
As an alternative to #300 I've created #318, which fixes the issue too.

**Comment by ignatovSA** [comment]
The fix is released in `1.4.3` version.
